src/server/pps/cmds/cmds.go

```diff
@@ -404,7 +404,7 @@ All jobs created by a pipeline will create commits in the pipeline's repo.
 					return err
 				}
 				if len(request.Inputs) != 0 {
-					fmt.Printf("WARNING: field `inputs` is deprecated, use `input` instead.\n")
+					fmt.Printf("WARNING: field `inputs` will be deprecated in v1.6. Both formats are valid for v1.4.6 to 1.5.x. See docs for the new input format: http://pachyderm.readthedocs.io/en/latest/reference/pipeline_spec.html \n")
```

Comment:
This warning message is a little inaccurate, normally a feature is called "deprecate" when it still works but isn't the recommended way to do something. It'd be more accurate to say: "`inputs` is deprecated and will be removed in v1.6..."